certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1850 @@
+        "summary": "List of all custom fields",
+        "description": "List of all custom fields",
+        "operationId": "List-of-all-custom-fields",
+        "parameters": [
+          {
+            "name": "pageNumber",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "AppToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "69a489d9-53fd-43aa-b04c-5c15ed4c73dd",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "UserToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "98ae7807-0902-4e66-9b0a-bfe0ecd204de",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/": {
+      "x-ms-notification-content": {
+        "schema": {
+          "type": "object",
+          "properties": {
+            "WebhookId": {
+              "type": "string",
+              "description": "WebhookId"
+            },
+            "Properties": {
+              "type": "object",
+              "properties": {
+                "Date": {
+                  "type": "string",
+                  "description": "Date"
+                }
+              },
+              "description": "Properties"
+            },
+            "Notifications": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "Action": {
+                    "type": "string",
+                    "description": "Action"
+                  },
+                  "IndividualId": {
+                    "type": "string",
+                    "description": "IndividualId"
+                  },
+                  "RecordNumber": {
+                    "type": "string",
+                    "description": "RecordNumber"
+                  },
+                  "FirstName": {
+                    "type": "string",
+                    "description": "FirstName"
+                  },
+                  "LastName": {
+                    "type": "string",
+                    "description": "LastName"
+                  },
+                  "EmailAddress": {
+                    "type": "string",
+                    "description": "EmailAddress"
+                  }
+                }
+              },
+              "description": "Notifications"
+            }
+          }
+        },
+        "description": "Individual Created"
+      }
+    },
+    "/api/v1/Customers/{pageNumber}": {
+      "get": {
+        "responses": {
+          "default": {
```

Comment:
@RjImpx Please 'replace' the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.